google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadConversionEnhancement.java

```diff
@@ -77,7 +88,9 @@ public static void main(String[] args)
       params.conversionActionId = Long.parseLong("INSERT_CONVERSION_ACTION_ID_HERE");
       params.orderId = "INSERT_ORDER_ID_HERE";
 
-      // Optional: Specify the user agent, restatement value and restatement currency code.
+      // Optional: Specify the conversion date/time, user agent, restatement value and restatement
```

Comment:
Nit: there should be a comma after `restatement value` I believe.

Reply:
Fixed